dbbackup/db/postgresql.py

```diff
@@ -57,9 +65,14 @@ def _restore_dump(self, dump):
 
         # without this, psql terminates with an exit value of 0 regardless of errors
         cmd += " --set ON_ERROR_STOP=on"
+
+        if self.schemas:
+            cmd += " -n " + " -n ".join(self.schemas)
+
         if self.single_transaction:
             cmd += " --single-transaction"
-        cmd += " {}".format(self.settings["NAME"])
+
+        cmd += f" {self.settings['NAME']}"
```

Comment:
There's several unnecessary f-string changes in this PR.

There's a ton of small changes like this that need to happen across the whole repo. Will leave that for a future refactoring PR.

Reply:
@Archmonger fixed.